<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgImportShapefile</class>
 <widget class="QDialog" name="dlgImportShapefile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>201</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>254</width>
    <height>133</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Import Shapefile</string>
  </property>
  <property name="toolTip">
   <string>Import assingments from a shapefile</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="lblDistField">
     <property name="text">
      <string>District &amp;Field</string>
     </property>
     <property name="buddy">
      <cstring>cmbDistField</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QgsFieldComboBox" name="cmbDistField">
     <property name="toolTip">
      <string>Shapefile attribute containing the district number</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblShapefile">
     <property name="text">
      <string>&amp;Shapefile</string>
     </property>
     <property name="buddy">
      <cstring>fwShapefile</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QgsFieldComboBox" name="cmbNameField">
     <property name="toolTip">
      <string>Shapefile attribute containing the district name</string>
     </property>
     <property name="allowEmptyFieldName">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QgsFileWidget" name="fwShapefile">
     <property name="dialogTitle">
      <string>Import Shapefile</string>
     </property>
     <property name="filter">
      <string>*.shp</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblNameField">
     <property name="text">
      <string>&amp;Name Field</string>
     </property>
     <property name="buddy">
      <cstring>cmbNameField</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblMembersField">
     <property name="text">
      <string>&amp;Member Field</string>
     </property>
     <property name="buddy">
      <cstring>cmbMembersField</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QgsFieldComboBox" name="cmbMembersField">
     <property name="toolTip">
      <string>Shapefile attribute containing the number of members in the district</string>
     </property>
     <property name="allowEmptyFieldName">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFieldComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsfieldcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsFileWidget</class>
   <extends>QWidget</extends>
   <header>qgsfilewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>dlgImportShapefile</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>dlgImportShapefile</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
